mock-server/app.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const cors = require('cors')
+const app = express()
+const port = 3001
+
+const SESSION_TOKEN = 'iamasessiontoken'
+
+const services = {
+    session: {
+        open: params => ({
+            status: 0,
+            data: [
+                SESSION_TOKEN,
+                params.username,
+                0
+            ]
+        }),
+        isalive: params => ({
```

Comment:
maybe we should put some minimal effort into these mock API requests (such as session&parameter checking).
This makes the mock server more useful for unit testing